docs/administrative-requirements.md

```diff
@@ -0,0 +1,129 @@
+
+Resources:
+
+* [Privacy Threshold Analysis (PTA) and Privacy Impact Assessment (PIA) Submittal Checklist and Process Overview](https://dvagov.sharepoint.com/sites/OITPrivacyHub/PTA%20Training%20Resources/Forms/AllItems.aspx?id=%2Fsites%2FOITPrivacyHub%2FPTA%20Training%20Resources%2FPTA%20and%20PIA%20Submittal%20Checklist%20and%20Process%20Overview%2Epdf&parent=%2Fsites%2FOITPrivacyHub%2FPTA%20Training%20Resources&isSPOFile=1&OR=Teams%2DHL&CT=1709826099209&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yNDAxMDQxNzUwNCIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D) (pdf, must be on VA network to view)
+* [PTA Training Resources](https://dvagov.sharepoint.com/sites/OITPrivacyHub/SitePages/Privacy-Impact-Assessment.aspx?csf=1&web=1&e=u8YJ0w) (VA SharePoint, must be on VA network to view)
+
+## ESECC
+
+Depending on the operating boundaries or ports used by the client system, a request to the Enterprise Security External Change Council (ESECC) may be needed to authorize an opening in the firewall to enable a connection between the client system and the Enterprise Event Bus.
+
+Prior to determining if your system will require an ESECC, teams are highly encouraged to first attempt connecting to the Enterprise Event Bus.
+
+![A diagram showing various possibilities and whether a system would likely need an ESECC. If you are on Amazon Web Services (AWS) inside VA Enterprise Cloud (VAEC), or on LHDI, no ESECC is needed. Instead, a Service Now ticket is needed. If you are on AWS outside VAE, or if you are on Non-AWS Cloud (e.g. Azure), you should check if ESECC is needed. If you are on Premise, e.g., VistA, an ESECC is needed.](img/Client-Environments-ESECC-Decision-Circles.svg)
+
+Resources:
+
+* [ESECC Public URL process (GitHub)](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/ESECC-Public-URL-process.md#prereq)
+* [Tutorial video](https://dvagov.sharepoint.com/:v:/r/sites/OITECOESDKM/KM%20video%20library/Stream%20Migrated%20Videos/Knowledge%20Management%20Videos/KB0106755%20-%20Network%20ESECC%20-%20Create%20a%20Change%20Request-20221206_034448.mp4?csf=1&web=1&e=cCAS5z&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D) on creating a Change Request in ESECC
+* [ESECC Self-service Portal](https://esecc.va.gov/CGWeb/Main.aspx) for initiating new requests or change requests
+
+## WASA Testing
+
+Web Application Security Assessment (WASA) scanning may be needed when a client system builds a web application for consuming events from the bus.
+
+The VA SAVD WASA Coordination team can be contacted via email at [VASAVDWASACoordination@va.gov](mailto:VASAVDWASACoordination@va.gov). A request for WASA testing can be submitted on the [Security Assessment Portal home page](https://portalapps.vansoc.va.gov/EAS/Home).
```

Comment:
Might want to note that a 30 day notice is required for WASA testing - build that into your schedule so that you are getting the request in at the right time.